onnxruntime/test/providers/cpu/tensor/resize_op_test.cc

```diff
@@ -1060,6 +1076,11 @@ TEST(ResizeOpTest, ResizeOpNearestUpSample_Floor_Align_Corners) {
 }
 
 TEST(ResizeOpTest, ResizeOpNearest_OneToOneMappingBetweenInputAndOutputDataDims) {
+  // TODO: Unskip when fixed #41968513
+  if (DefaultDmlExecutionProvider().get() != nullptr) {
+    GTEST_SKIP() << "Skipping because of the following error: The difference between expected[i] and output[i] is 3, which exceeds threshold";
```

Comment:
Seems a real issue given a difference of 3, rather than just tolerance matters. Support for ceil mode is newer. Probably should split 41968513 into multiple separate bugs as you find more.

We only have 1 test case in the ONNX conformance suite for `tf_half_pixel_for_nn`.

```json
    {
      "graph_name": "Resize-11 2D linear 2x tf_half_pixel_for_nn",
      "op_type": "Resize",
      "version": 11,
      "mode": "linear",
      "coordinate_transformation_mode": "tf_half_pixel_for_nn",
      "X": [[1, 2],
            [3, 4]],
      "roi": [0,0,1,1],
      "scales": {"value": [2, 2], "use_variable_length_values": true},
      "Y": [[1.75, 2.25, 2.5, 2.5],
            [2.75, 3.25, 3.5, 3.5],
            [3.25, 3.75, 4.0, 4.0],
            [3.25, 3.75, 4.0, 4.0]],
      "T1": "float32",
      "T2": "float32",
      "verification": {"ulp": 8 /* 0.000001 RTOL */}
    },
```